xcube/core/gen2/error.py

```diff
@@ -20,6 +20,6 @@
 # SOFTWARE.
 
 
-class GenError(Exception):
+class CubeGeneratorError(ValueError):
```

Comment:
Can we say with certainty that every CubeGeneratorError is a ValueError?

Reply:
It is common to use Value Error as base class for derived errors as ValueError is generic (see zarr, xarray and others). It helps handling errors, if a derived error is one of the base errors.

Reply:
ok